test/unit/Axes.spec.js

```diff
@@ -301,7 +302,52 @@ describe("Axes", function () {
       input2.destroy();
       input3.destroy();
     });
+
+    it("should work with pan gesture when connecting PanInput after PinchInput disconnected.", done => {
+      // Given
+      const MOVE_HORIZONTALLY = {
+        pos: [0, 0],
+        deltaX: 100,
+        deltaY: 10,
+        duration: 200,
+        easing: "linear"
+      };
+      const MOVE_VERTICALLY = {
+        pos: [0, 0],
+        deltaX: 0,
+        deltaY: 100,
+        duration: 200,
+        easing: "linear"
+      };
+      const target = document.querySelector("#sandbox");
+      const pinchInput = new PinchInput(target, {inputType: ["touch"]});
+      const panInput = new PanInput(target, {inputType: ["touch"]});
+
+      this.inst.connect("x", pinchInput);
+      this.inst.disconnect(pinchInput);
+      this.inst.connect(["x", "otherX"], panInput);
+
+      const prevX = this.inst.get()["x"];
+      const prevY = this.inst.get()["otherX"];
+
+      // When // pan 제스춰 동작 시 동작해야함 
+			Simulator.gestures.pan(target, MOVE_HORIZONTALLY, () => {
+				Simulator.gestures.pan(target, MOVE_VERTICALLY, () => {
+          // Then
```

Comment:
Test is gooood!

Indentation looks not beautiful and it's better to use english than korean

![image](https://user-images.githubusercontent.com/3255471/38188142-76af824c-3695-11e8-82d9-065d7bee478a.png)